docs/foundation/Icons/Usage.stories.tsx

```diff
@@ -67,7 +76,7 @@ DecorativeIcon.parameters = {
 };
 
 export const SemanticIcon = () => (
-  <Level4 role="img" title="Warning!" iconSize="M" semantic="negative" />
+  <Level4 role="img" title="Warning!" iconSize="M" color="negative" />
```

Comment:
Given the sample above, do we need to maintain this one?

Reply:
I'll keep this one and remove the others 👍